lib/Client.js

```diff
@@ -601,8 +602,11 @@ class Client extends EventEmitter {
                 };
             } else if(content.content !== undefined && typeof content.content !== "string") {
                 content.content = "" + content.content;
-            } else if(content.content === undefined && !content.embed && !file) {
-                return Promise.reject(new Error("No content, file, or embed"));
+            } else if(content.content === undefined && !content.embed && !content.embeds && !file) {
+                return Promise.reject(new Error("No content, file, or embeds"));
+            } else if(content.embed) {
```

Comment:
Check if anything exists in `embeds` to prevent accidental overriding

Reply:
What'd you exactly meant by "prevent accidental overriding"??

Reply:
Right now if someone provides `embed` and `embeds`, then `embeds` will be overwritten

Reply:
Do you want if someone provides both `embed` and `embeds`, it'll send both or only `embeds`? 😅

Reply:
You'd want it to add embed to embeds rather then override I think no?

Reply:
If someone provides both `embed` and `embeds`, send both and let Discord handle that
FI `embeds` will override `embed` if it is supplied